wsd/Storage.cpp

```diff
@@ -832,6 +832,8 @@ WopiStorage::WOPIFileInfo::WOPIFileInfo(const FileInfo& fileInfo,
             anonObject->set("UserFriendlyName", COOLWSD::anonymizeUsername(_username));
         }
         anonObject->stringify(wopiResponse);
+
+        // coverity[freed_arg] - object/anonObject's nRefCount is > 1 here
```

Comment:
The trouble here is that both object and anonObject goes out of scope, but it's fine, the first will decrease nRefCount to 1, the second will decrease to 0 & free, so there is no double-free.